samples/bluetooth/mesh/sensor_client/src/model_handler.c

```diff
@@ -23,15 +23,15 @@ static void sensor_cli_data_cb(struct bt_mesh_sensor_cli *cli,
 		if (value->val1) {
 			printk("Presence detected\n");
 		} else {
-			printk("End of presence\n");
+			printk("No presence detected\n");
```

Comment:
Note that this change will brake current system tests for the sensor client sample.